dist/MANIFEST.txt

```diff
@@ -1,39 +0,0 @@
-

```

Comment:
Even when telling Grafana we're ok with this plugin being unsigned, if the files are modified, it won't load the plugin.